<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="inputMLabel">
         <property name="text">
          <string>Введите M</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="inputMSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="inputNLabel">
         <property name="text">
          <string>Введите N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="inputNSpinBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="doPushButton">
         <property name="text">
          <string>Do It!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
